src/_includes/head.html

```diff
@@ -34,24 +37,105 @@
     <script async defer src="https://buttons.github.io/buttons.js"></script>
   {% endif %}
 
-  <script>
-      window.addEventListener('scroll', function (e) {
-          var body       = document.getElementsByTagName('body')[0];
-          var siteHeader = body.querySelector('.site-header');
+  {% if isDocumentation %}
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jquery.perfect-scrollbar/0.6.12/css/perfect-scrollbar.min.css">
+    <script defer type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery.perfect-scrollbar/0.6.12/js/perfect-scrollbar.min.js"></script>
+    <script>
+        function initNav(nav) {
+            var MIN_BODY_WIDTH = 900;
+            var FOOTER_HEIGHT  = 63;
+            var HEADER_HEIGHT  = 52;
 
-          if (!siteHeader)
-              return;
+            function updatePosition() {
```

Comment:
Moreover you don't need script for this behavior. Just put
```javascript
<div class="push"></div>
<footer class="site-footer">
<div class="site-footer-content">
<div class="footer-col-1">
	<p><strong>TestCafe</strong>: Automated browser testing for the modern web development stack<br>
	Copyright © 1998-2016 Developer Express Inc.</p>
</div>
</div>

```
a the end of ```<article>``` and add ```position: absolute, left: 0, right: 0``` to ```.site-footer```. Also you can add  ```min-width ``` for  ```.site-footer```